templates/cisco_nxos_show_forwarding_ipv4_route.template

```diff
@@ -0,0 +1,22 @@
+Value PREFIX (\d+\.\d+\.\d+\.+\d+\/\d+)
+Value Required,List NEXTHOP (Drop|Receive|Attached|\d+\.\d+\.\d+\.\d+)
+Value List INTERFACE (\S+)
+
+Start
+  ^Prefix.*Partial\s+Install\s*$$ -> FIB
```

Comment:
Do you mind updating the header to match at least each word within the header?

Reply:
Modified to be more explicit.